tools/WORLD/src/matlabfunctions.cpp

```diff
@@ -238,25 +240,38 @@ void interp1Q(double x, double shift, const double *y, int x_length,
   delete[] delta_y;
 }
 
+// You must not use these variables.
+// Note:
+// I have no idea to implement the randn() and randn_reseed() without the
+// global variables. If you have a good idea, please give me the information.
+static uint32_t g_randn_x = 123456789;
+static uint32_t g_randn_y = 362436069;
+static uint32_t g_randn_z = 521288629;
+static uint32_t g_randn_w = 88675123;
+
+void randn_reseed(void) {
+    g_randn_x = 123456789;
+    g_randn_y = 362436069;
+    g_randn_z = 521288629;
+    g_randn_w = 88675123;
+}
+
 double randn(void) {
-  static unsigned int x = 123456789;
-  static unsigned int y = 362436069;
-  static unsigned int z = 521288629;
-  static unsigned int w = 88675123;
-  unsigned int t;
-  t = x ^ (x << 11);
-  x = y;
-  y = z;
-  z = w;
-
-  unsigned int tmp = 0;
-  for (int i = 0; i < 12; ++i) {
-    t = x ^ (x << 11);
-    x = y;
-    y = z;
-    z = w;
-    w = (w ^ (w >> 19)) ^ (t ^ (t >> 8));
-    tmp += w >> 4;
+  uint32_t t;
+  t = g_randn_x ^ (g_randn_x << 11);
+  g_randn_x = g_randn_y;
+  g_randn_y = g_randn_z;
+  g_randn_z = g_randn_w;
+  g_randn_w = (g_randn_w ^ (g_randn_w >> 19)) ^ (t ^ (t >> 8));
```

Comment:
This does not match with mmorise's commit, is there a reason?
https://github.com/mmorise/World/commit/9a26692aff144db923daa7e6c75404ea6c9d9cbd#diff-187e775cb821482add4032722c5e716aR262